dask_cuda/proxify_host_file.py

```diff
@@ -583,25 +583,44 @@ def oom(nbytes: int) -> bool:
                 mr = rmm.mr.FailureCallbackResourceAdaptor(current_mr, oom)
                 rmm.mr.set_current_device_resource(mr)
 
+    def evict(self) -> int:
+        """Manually evict 1% of host limit.
```

Comment:
I understand this was already like this before and not quite sure how useful that would be, but should the 1% value be configurable somehow?

Reply:
I don't think that is necessary. Hopefully, we get to a point where Dask never calls this.